src/engine/source/builder/builders/stageBuilderNormalize.cpp

```diff
@@ -53,7 +53,7 @@ Builder getStageNormalizeBuilder(std::shared_ptr<Registry> registry)
                 if (!block.isObject())
                 {
                     throw std::runtime_error(fmt::format(
-                        "Invalid array item type, expected \"object\" but got \"{}\"",
+                        R"(Invalid array item type, expected "object" but got "{}")",
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices

Raw strings